### PR TITLE
Quarkus_cli re-start application scenario

### PR DESCRIPTION
**Pull request by pjgg**

### CODE REVIEW

**Comment by kshpak** [comment]
@pjgg provide some description pls, it would be easier to grasp

**Comment by pjgg** [comment]
> @pjgg provide some description pls, it would be easier to grasp

Quarkus-cli scenario failed last week due to an error in upstream that this week seems to be fixed(or at least we can't not reproduce with the same test). This PR will rollback the changes that we did last week where we created a new test to cover this scenario and do a walkaround to bypass this error in an existing scenario.

If tomorrow the daily build pass we can close the issue in upstream

File: quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java [hunk 1]
```diff
@@ -115,7 +112,7 @@ public void shouldAddAndRemoveExtensions() throws InterruptedException {
         assertTrue(result.isSuccessful(), SMALLRYE_HEALTH_EXTENSION + " was not uninstalled. Output: " + result.getOutput());
 
         // The health endpoint should be now gone
-        startAfter(app, Duration.ofSeconds(CMD_DELAY_SEC));
```
Comment (jsmrcka) [inline quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java:118]: There are unused constants left in this class: `CMD_DELAY_SEC` and `SMALLRYE_OPENAPI`. Plus the method `shouldAddAndRemoveExtensions` declares an `InterruptedException` which is never being thrown.

Maybe just do a revert of https://github.com/quarkus-qe/quarkus-test-suite/pull/361, that should take care of everything.

**Review by jsmrcka — CHANGES_REQUESTED** [review]

**Comment by jsmrcka** [comment]
Please rename the PR/commit to make it clear it reverts the https://github.com/quarkus-qe/quarkus-test-suite/pull/361 and link it in the description.

**Review by jsmrcka — CHANGES_REQUESTED** [review]
See https://github.com/quarkus-qe/quarkus-test-suite/pull/370#issuecomment-961636812

**Review by jsmrcka — APPROVED** [review]

**Pull request merged**